satellite/accounting/bucketstats.go

```diff
@@ -5,26 +5,18 @@ package accounting
 
 import (
 	"github.com/skyrings/skyring-common/tools/uuid"
-	monkit "gopkg.in/spacemonkeygo/monkit.v2"
-
-	"storj.io/storj/pkg/pb"
 )
 
-var mon = monkit.Package()
-
 // BucketTally contains information about aggregate data stored in a bucket
 type BucketTally struct {
 	ProjectID  uuid.UUID
 	BucketName []byte
 
 	ObjectCount int64
 
-	Segments        int64
-	InlineSegments  int64
-	RemoteSegments  int64
-	UnknownSegments int64
```

Comment:
i guess we don't use `UnknownSegments` for anything? I wonder why it was there in the first place.

Reply:
It was a default case when the segment type wasn't specified, however that would mean our database is corrupt.